packages/core/test/WorkflowExecute.test.ts

```diff
@@ -191,7 +191,8 @@ describe('WorkflowExecute', () => {
 						return nodeData.data.main[0];
 					});
 
-					expect(resultData).toEqual(testData.output.nodeData[nodeName]);
+					// used toMatchObject instead of toEqual to ignore paired items
+					expect(resultData).toMatchObject(testData.output.nodeData[nodeName]);
```

Comment:
Maybe we should keep toEqual, this might loosen tests too much. Instead you could delete `pairedItem` from the resultData?

Reply:
I tried that, but the same test is written for paired items tests which doesn't make sense if I removed the paired items.
I will extract this test and remove the key as you suggested